### DISCUSSION TITLE
FileNotFoundError: [WinError 2] The system cannot find the file specified

### DISCUSSION THREAD

**Discussion by Vs-kishu** [Atri-Labs/atrilabs-engine#598] [Help: Installation & Start] [answered]
<img width="960" alt="Screenshot 2022-12-21 173310" src="https://user-images.githubusercontent.com/91000916/208901309-ea3fc33c-a441-42d5-b24a-5b6dcbce397c.png">
<img width="960" alt="Screenshot 2022-12-21 173249" src="https://user-images.githubusercontent.com/91000916/208901314-3cc9d3d3-cd42-4a2e-86b6-8ab9a96b9f22.png">

**Comment by darshitac11** [comment, marked as answer]
@Vs-kishu - follow the answer shared here: https://github.com/Atri-Labs/atrilabs-engine/discussions/342

**Comment by Vs-kishu** [comment]
can't able to resolve issue with this solution can you help me out by
screen share
